pkg/math/oracle/utils.go

```diff
@@ -55,3 +56,81 @@ func (m *MedianAggregator) GetProviderPrice(
 
 	return scaledPrice, nil
 }
+
+// ValidateMarketMap validates the market map configuration and its expected configuration for
+// this aggregator. In particular, this will
+//
+//  1. Ensure that the market map is valid (ValidateBasic). This ensure's that each of the provider's
+//     markets are supported by the market map.
+//  2. Ensure that each path has a corresponding ticker.
+//  3. Ensure that each path has a valid number of operations.
+//  4. Ensure that each operation has a valid ticker and that the provider supports the ticker.
+func ValidateMarketMap(
```

Comment:
Since this is the only aggregator we are going to support AND this function does not rely on anything aggregator-specific AFAIK, why not just include this in mm.ValidateBasic()?

Reply:
separating out only makes sense to me if the plan is to support multiple aggregation strategies

Reply:
I prefer deferring whether we want this to be the canonical one to a later time, I can see us porting over the logic in the future.

Reply:
Makes sense - holding over in case we wanna factor out.  I still think this should live in `x/marketmap/types` as a function called inside of `ValidateBasic`.  In the event that we want to move it elsewhere, it will be easy, but as things currently are, this encompasses the full validation for what can be on and off chain